### ISSUE TITLE
Higher Quality Versions of the Battle Mode tracks were found

### ISSUE THREAD

**Issue by letsgoawaydev** [DBTDerpbox/LEB-Resources#49]
check the pinned comment under this video https://youtu.be/Imt1rM8WDCo?si=jCyMoP1-wyOXyBOE

From @unreleasedlce on YouTube:
 
> a higher quality version has been found
> these have been found by mattnl & brought to my attention to cluck nugget, so massive thanks to them
> i've uploaded the files here for anyone interested: [https://www.mediafire.com/folder/930155pmfra8m/MG01](https://www.mediafire.com/folder/930155pmfra8m/MG01)

**Comment by DBTDerpbox** [comment]
yeah someone in the lem discord noticed this and got some of the songs but never finished it

**Comment by DexrnZacAttack** [comment]
> yeah someone in the lem discord noticed this and got some of the songs but never finished it

Lol and I forgot where I was at... Did I ever give the files?
